Chatkit/Authentication/Token Providers/TestTokenProvider.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import PusherPlatform
+
+/// TestTokenProvider retrieves tokens from the Chatkit service's Test Token Provider, which
+/// is for development user only, and must be enabled for your instance in the Chatkit Dashboard.
+///
+/// The test token provider will always sign a token for the requested userID, without applying any
+/// form of authentication.
+public class TestTokenProvider: TokenProvider {
+    
+    // MARK: - Properties
+    
+    private static let userIdentifierQueryItemName = "user_id"
+    private static let urlScheme = "https"
+    private static let urlHost = "pusherplatform.io"
+    private static let urlService = "services/chatkit_token_provider"
+    private static let urlResource = "token"
+    
+    /// The locator for your instance, the same value from the Dashboard which you use to construct
+    /// the Chatkit object.
+    public let instanceLocator: String
+    
+    /// The userID to fetch tokens for. A token will always be signed for this userID without any
```

Comment:
"The userID for whom to fetch tokens" maybe reads a little better I think (this is repeated below also)